frontend/index.html

```
@@ -49,6 +49,10 @@ <h1 class="brand">
             <input type="radio" name="source-mode" value="manual" />
             <span>手動2点</span>
           </label>
+          <label class="mode-chip">
+            <input type="radio" name="source-mode" value="loop" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)